pypeit/par/pypeitpar.py

```diff
@@ -1698,9 +1698,9 @@ class SlitMaskPar(ParSet):
 
 
     """
-    def __init__(self, obj_toler=None, assign_obj=None, nsig_thrshd=None,
+    def __init__(self, obj_toler=None, assign_obj=None, skip_serendip=None, nsig_thrshd=None,
                  slitmask_offset=None, use_dither_offset=None, bright_maskdef_id=None, extract_missing_objs=None,
-                 use_alignbox=None):
+                 missing_objs_fwhm=None, missing_objs_boxcar_rad=None, use_alignbox=None):
```

Comment:
I don't see why these parameters missing_objs_fwhm and missinb_objs_boxcar_rad are specific to coadd2d. They are not. If you want to be able to specify these things that is a general question for normal single frame reductions. These parameters should thus be added to the parsers for objfind or extract and not for coadd2d.